src/qml/ChatPage.qml

```diff
@@ -79,66 +75,108 @@ Kirigami.Page {
 			model: kaidan.messageModel
 
 			delegate: ChatMessage {
+				width: parent.width
 				sentByMe: model.recipient !== kaidan.jid
 				messageBody: model.message
 				dateTime: new Date(timestamp)
 				isRead: model.isDelivered
 				recipientAvatarUrl: kaidan.avatarStorage.getHashOfJid(author) !== "" ?
-						    kaidan.avatarStorage.getAvatarUrl(author) :
-						    kaidan.getResourcePath("images/fallback-avatar.svg")
+									kaidan.avatarStorage.getAvatarUrl(author) :
+									kaidan.getResourcePath("images/fallback-avatar.svg")
```

Comment:
This is the exact problem with using tabs in different indentation lengths: My solution for this would be to only use the tabs till the beginng of "recipientAvatarUrl:", then use a fixed number of spaces, so it's the same in all indentation styles and always looks as wanted.

Reply:
I'm sorry to interrupt your idea, but we could JUST USE SPACES for QML; for me, that would solve 100% of formating issues.

Reply:
The advantage is that the display style is separated from the actual code. The editor also doesn't have to find out which indentation length is used in the file, it just takes the length preferred by the user.